Assets/Scripts/UI/InGame/InGameMenu.cs

```diff
@@ -5,45 +5,44 @@ namespace Assets.Scripts.UI.InGame
     public class InGameMenu : MonoBehaviour
     {
         public GameSettingMenu GameSettingsMenu;
+
         public GraphicSettingMenu GraphicSettingsMenu;
-        public void ShowGameSettingsMenu()
+
+        // Used by Button.
```

Comment:
I think comments like these are useful, since I was about to make this `private`, thinking it was a Unity message.